src/alire/alire.ads

```diff
@@ -55,6 +55,22 @@ package Alire with Preelaborate is
    --  This type is used to ensure that folder separators are externally always '/',
    --  and internally properly converted to the platform one
 
+   --  To clarify constants/functions declared herein:
+
+   subtype Absolute_File is Platform_Independent_Path;
+   --  Filenames with full path
+
+   subtype Absolute_Path is Platform_Independent_Path;
```

Comment:
What is the difference between Absolute_File and Absolute_Path? (same question for Relative_*)

Reply:
The intent is that *_File is a regular file and *_Path is a folder. Perhaps the latter should be renamed to *_Folder (or *_Directory)